src/stories/components/BigNumber/BigNumber.story.tsx

```diff
@@ -126,45 +179,49 @@ SparklineAndIconVertical.args = {
 	label: 'Visitors',
 };
 
-const CurrencyFormat = bindWithProps(BigNumberStory);
+const SparklineMethodLast = bindWithProps(BigNumberStory);
+SparklineMethodLast.args = {
+	dataKey: 'y',
+	children: [
+		<Line key="line" dimension="x" metric="y" scaleType="linear" />,
+	],
+	orientation: 'vertical',
+	label: 'Visitors'
+}
+
+const CurrencyFormat = bindWithProps(CurrencyFormatStory);
 CurrencyFormat.args = {
 	children: undefined,
-	dataKey: 'x',
+	dataKey: 'value',
 	orientation: 'horizontal',
 	label: 'Ad Spend',
 	numberFormat: '$,.2f',
 };
 
-const PercentageFormat = bindWithProps(BigNumberStory);
+const PercentageFormat = bindWithProps(PercentageFormatStory);
 PercentageFormat.args = {
 	children: undefined,
-	dataKey: 'x',
+	dataKey: 'value',
 	orientation: 'horizontal',
 	label: 'Capacity',
 	numberType: 'percentage',
 };
 
-const CompactFormat = bindWithProps(BigNumberStory);
+const CompactFormat = bindWithProps(CompactFormatStory);
 CompactFormat.args = {
 	children: undefined,
-	dataKey: 'x',
+	dataKey: 'value',
 	orientation: 'horizontal',
 	label: 'Requests',
 };
 
-// const NullData = bindWithProps(BigNumberStory);
```

Comment:
Could this case still exist?  If type safety helps us make data never null, then it's probably safe

Reply:
Yeah we can remove both of these stories, my bad though as I had commented them out.

Reply:
OK - in that case, since data can't ever be null as it is  a required property of Chart - do you all think we need it?